README.md

```diff
@@ -58,6 +58,11 @@ Additionally, the Hive plugin must be configured with location of your Hive meta
 
     -Dhive.metastore.uri=thrift://localhost:9083
 
+#### Common problems
+In newer IntelliJ versions the "VM options" field in the Run/Debug configuration may not be present by default. You can add it by clicking  "Modify options" and selecting "Add VM options". It is common to accidentally copy the VM options into the "Program arguments" field. Doing so results in errors on startup that start with "Explicit bindings are required ...". 
```

Comment:
```suggestion
* In newer IntelliJ versions the "VM options" field in the Run/Debug configuration may not be present by default. You can add it by clicking  "Modify options" and selecting "Add VM options". 

* If an error similar to the following is displayed at startup:
"Explicit bindings are required ..."
 This can be caused by including the VM options into the "Program arguments" field in IntelliJ. Find the "Program arguments" field, and if the VM options in [Overview](https://github.com/prestodb/presto#overview) are present, remove the VM options.
```
Suggest separating the two problems as items in an unordered list. 
Suggest reversing the order in the second problem, to present the user experience of the error, and what to do if to solve the problem.